internal/test/e2e/plugins.go

```diff
@@ -279,6 +279,8 @@ var VersionCompatibilityOSSPluginConfigurationTests = []VersionCompatibilityPlug
 		ConfigureForService: true,
 		ConfigureForRoute:   true,
 	},
+	// DP < 3.1:
```

Comment:
While you are at it, would you mind documenting the expected changes for older DPs too? Like that `redis_*` fields are removed for DP < 2.7 etc

Reply:
[cfa8dd2](https://github.com/Kong/koko/pull/515/commits/cfa8dd288786e9cb4951be2d4f6a3e007e9e5a62)

Also noticed that `redis_password` is not being removed? https://github.com/Kong/koko/blob/69228016c6c464471b8ea53bb5426677e073f2ff/internal/server/kong/ws/config/compat/compatibility_table.go#L382-L398

Reply:
that seems correct. `redis_password` was added in a previous kong version. ref: https://github.com/Kong/kong/pull/8032